Content.Client/Verbs/UI/VerbMenuPresenter.cs

```diff
@@ -182,8 +182,12 @@ public override void OnKeyBindDown(ContextMenuElement element, GUIBoundKeyEventA
                 if (verbElement.SubMenu == null || verbElement.SubMenu.ChildCount == 0)
                     return;
 
-                if (verbElement.SubMenu.MenuBody.Children.First() is not VerbMenuElement verbCategoryElement)
+                if (verbElement.SubMenu.MenuBody.ChildCount != 1
+                    || verbElement.SubMenu.MenuBody.Children.First() is not VerbMenuElement verbCategoryElement)
+                {
+                    verbElement.SubMenu.Visible = true;
```

Comment:
If you just set visible directly, it will cause issues with the various on-mouse-entered & existed stuff because the sub-menu is not in the menu-stack. You need to use `OpenSubMenu(verbElement)`.

There's also a separate issue here where after the click, the top-most popup looses focus and auto closes (though this doesn't happen if it wasn't properly opened).

If you want I can add a commit to fix that, along with some jank related to `CancellationTokenSource`s

Reply:
Feel free to push the changes.

Reply:
Done

Reply:
Alright. Looks good from what I can see looking at the code.